### DISCUSSION TITLE
Is a paid developer account needed?

### DISCUSSION THREAD

**Discussion by gurubac** [sparkle-project/Sparkle#2310] [Q&A] [answered]
Hey, I'm sure you have gotten this question before, but I cannot find a super clear answer so I am asking it here. Do I need to have a paid apple developer account in order to use sparkle and distribute updates to users?

**Comment by zorgiepoo** [comment, marked as answer]
Sparkle does not require using a paid Apple developer ID account, no.

if you want to distribute your app on your website without requiring users to go into additional settings / workflows around Gatekeeper to open your app then yes creating a signed and notarized developer ID app costs some money from Apple. But this is unrelated to Sparkle.

**Reply by gurubac** [reply]
Do you know what the exact "additional settings/workflows around Gatekeeper" are? Like is it getting the user to codesign manually, do some chmod/xattr stuff, or just pressing open anyway in security and privacy settings? If its just the last option I am fine with that and will try to use Sparkle.

**Reply by zorgiepoo** [reply]
Open anyway. System settings is the long way. Right click and open if app doesn’t launch is short way https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac

**Comment by gurubac** [comment]
Sounds great, thank you for help!
